classes/core/PKPApplication.php

```diff
@@ -173,6 +173,7 @@ class_alias('\PKP\payment\QueuedPayment', '\QueuedPayment'); // QueuedPayment in
         }
 
         // Ensure that nobody registers for hooks that are no longer supported
+        Hook::addUnsupportedHooks('CitationDAO::afterImportCitations'); // pkp/pkp-lib#10692 Removed with introduction of structured citations
```

Comment:
This hook does not exist any more, we removed it during the DB preparation for structured citations, s. https://github.com/pkp/pkp-lib/issues/11238